addons/hr_presence/models/hr_employee.py

```diff
@@ -10,50 +10,44 @@
 _logger = logging.getLogger(__name__)
 
 
-class ResCompany(models.Model):
-    _inherit = 'hr.employee'
+class Employee(models.AbstractModel):
+    _inherit = 'hr.employee.base'
 
-    hr_presence_state = fields.Selection([
```

Comment:
So hr_presence_state has become a non stored selection field ? 

I'm not sure it's correct.